assets/translations/Portuguese(Brazil)

```diff
@@ -68,8 +68,8 @@ Edit=Editar
 emus=emus
 Exit=Sair
 Explorer=Explorador
-favourites=
-File containing a list of aliases for the selector=Arquivo que contém a lista de alias para o Seletor 
+favourites=favoritos
+File containing a list of aliases for the selector=Arquivo que contém a lista de nomes alternativos para o seletor 
```

Comment:
be consistent with "alias" naming, because previously you used "Aliases=Outros termos"